pyaerocom/aeroval/experiment_output.py

```diff
@@ -566,7 +566,14 @@ def _create_statistics_json(self):
         stats_info = statistics_defaults
         stats_info.update(extended_statistics)
         if self.cfg.statistics_opts.add_trends:
-            stats_info.update(statistics_trend)
+            if self.cfg.processing_opts.obs_only:
+                obs_statistics_trend = {}
+                for key in statistics_trend.keys():
+                    if "mod" not in key:
+                        obs_statistics_trend[key] = statistics_trend[key]
+                stats_info.update(obs_statistics_trend)
```

Comment:
you could write it as a dictionary comprehension
``` python
            if self.cfg.processing_opts.obs_only:
                obs_statistics_trend = {
                    key: val
                    key, val in statistics_trend.items()
                    if "mod" not in key
                }
                stats_info.update(obs_statistics_trend)                    
```

Reply:
I initially did this, but pylance told me that I needed an else statement there as well

Reply:
strange...